src/api/comments.ts

```diff
@@ -0,0 +1,37 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
```

Comment:
you can import BASE_URL from api.ts to all places, where you need to use it instead of declarating it again